tests/test_crypto.py

```diff
@@ -4454,3 +4484,12 @@ def test_does_not_contain(self):
             [get_elliptic_curve(self.curve_factory.another_curve_name)]
         )
         assert curve not in curves
+
+
+def test_utf8_pem():
+    """
+    Read a certificate and raises an error in case of UTF8 fixture.
```

Comment:
This comment is incorrect.

Reply:
Hi reaperhulk! 

Maybe something like this?

Checks if an UTF-8 certificate raises an encoding exception.

Reply:
"Checks that a PEM encoded file with UTF-8 characters outside the PEM boundary does not raise an exception"